src/hooks/usePreGigLogic.ts

```
@@ -58,6 +63,56 @@ export interface PreGigLogicReturn {
+
+  const handleUpdateMerchPrice = useCallback(
+    (merchKey: string, newPrice: number) => {
+      updateBand(prevBand => ({
+        ...prevBand,
+        merchPrices: {
+          ...(prevBand.merchPrices || {}),
+          [merchKey]: newPrice
+        }
+      }))
+    },
+    [updateBand]
```

Biome (2.4.15)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)